deployment/lambdas/cf-url-rewriter/cf-url-rewriter.ts

```diff
@@ -0,0 +1,14 @@
+import { CloudFrontRequest, CloudFrontRequestCallback, CloudFrontRequestEvent, Context } from 'aws-lambda';
```

Comment:
Can you move this file under `deployment/lib/` folder? The folder structure is what is recommended by aws-cdk and easier to maintain too https://cdkworkshop.com/20-typescript/20-create-project/300-structure.html

Reply:
The stack code with the Lambda function is indeed under `lib` https://github.com/opensearch-project/opensearch-build/pull/1660/files#diff-7b903cefe6c331b7992ffa4ce8ba2bfe44772c0d451317d3a374a7b12f0bada6

Reply:
I was recommending to put `lambdas` under `lib` too

Reply:
If you expand more on the examples provided your link https://github.com/aws-samples/aws-cdk-intro-workshop/tree/master/code/typescript/main-workshop, they do put lambda as parallel dir as lib and bin.

Reply:
Okay sure, Thanks!